### PR TITLE
add some doc for notifications plugins

### PR DESCRIPTION
**Pull request by sabban**
add some doc for notifications plugins debugging

### CODE REVIEW

File: crowdsec-docs/docs/notification_plugins/intro.md [hunk 1]
```diff
@@ -612,3 +612,23 @@ Teams webhook
     ]
  } 
 ```
+
+### Debugging notifications plugins
+
+cscli cli tool provide some useful command to help write notification
+plugin configuration. Those are provided by the `cscli notifications`
+command a,d its subcommands.
+
```
Comment (LaurenceJJones) [inline crowdsec-docs/docs/notification_plugins/intro.md:621]: meant to be and?

Comment (sabban) [reply crowdsec-docs/docs/notification_plugins/intro.md:621]: yes

File: crowdsec-docs/docs/notification_plugins/intro.md [hunk 2]
```diff
@@ -612,3 +612,23 @@ Teams webhook
...
+plugin configuration. Those are provided by the `cscli notifications`
+command and its subcommands.
+
+First `cscli notifications list` will list the active configured
+notifications plugins. Then it's possible to get configuration
+information for each notification plugin using its given name by
+executing `cscli notifications inspect <name>`.
+
+The last other helpful subcommand is for testing notifications plugins
+directly. It takes an alert and reinject it to notifications subsystem
```
Comment (LaurenceJJones) [inline crowdsec-docs/docs/notification_plugins/intro.md:628]: Replace `It takes an alert and reinject it to notifications subsystem
through profiles like notifications usually do. So if the alert
actually matches the profile configuration, it goes through the
configured notifications plugins.`
With
`It takes an id and reinject the alert through the matched profile notifications. If the alert matches a profile with no configured notifications then no notification will be generated.`

**Review by LaurenceJJones — APPROVED** [review]
LGTM

**Pull request merged**